pydevices/acq400Devices/acq1001_st.py

```diff
@@ -24,11 +24,9 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
-try:
-    acq400_base = __import__('acq400_base', globals(), level=1)
-except:
-    acq400_base = __import__('acq400_base', globals())
+import importlib
```

Comment:
is importlib available for all pythons versions? then we could use it everywhere as a replacement for _minport